client/server/pwa/manifest.js

```diff
@@ -2,10 +2,62 @@
  * External dependencies
  */
 import querystring from 'querystring';
+import config from 'config';
```

Comment:
This is an internal dependency, I believe?

Reply:
yes it is 😛